### PR TITLE
fix dead backlink

### PR DESCRIPTION
**Pull request by punkle**

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -81,6 +81,11 @@
   to = "/docs/details/allowlisting-roadie-traffic/"
   force = true
 
+[[redirects]]
+  from = "/docs/getting-started/openapi-specs"
+  to = "/docs/api"
```
Comment (dtuite) [inline netlify.toml:86]: We should be redirecting to the trailing slash version of this. Otherwise it will just redirect again. eg https://roadie.io/docs/api

```suggestion
  to = "/docs/api/"
```

**Review by dtuite — CHANGES_REQUESTED** [review]

**Comment by punkle** [comment]
@dtuite done

**Review by sblausten — APPROVED** [review]

File: netlify.toml [hunk 2]
```diff
@@ -81,6 +81,11 @@
   to = "/docs/details/allowlisting-roadie-traffic/"
   force = true
 
+[[redirects]]
+  from = "/docs/getting-started/openapi-specs"
+  to = "/docs/api/authorization/"
```
Comment (dtuite) [inline netlify.toml:86]: Actually, this doesn't make any sense. 

The broken page is about how to add API specs to the catalog. The page you're replacing it with is about authenticating with the Roadie APIs.

These are completely unrelated. You need to find a different page to redirect to.

Comment (punkle) [reply netlify.toml:86]: Will I go back to what i had?

**Review by dtuite — APPROVED** [review]
